pkg/graph/resolvers/model_plan.go

```diff
@@ -108,9 +120,62 @@ func ModelPlanCreate(ctx context.Context, logger *zap.Logger, modelName string,
 		return nil, err
 	}
 
+	if emailService != nil && emailTemplateService != nil {
+		err = sendModelPlanCreatedEmail(
+			ctx,
+			emailService,
+			emailTemplateService,
+			emailService.GetConfig().GetDevTeamEmail(),
+			createdPlan,
+		)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Do we want to throw an error here and block? Wondering if, since this isn't in Faktory yet, it makes more sense to just async this call and log the error if we get it, but otherwise just pass through this function and let the GQL mutation respond before the email is sent

Reply:
Good point. In the course of development I ran into this error and it was disorienting, I will make this change.